security/tpm_linux_bios_measurements.ksy

```diff
@@ -0,0 +1,434 @@
+meta:
+  id: tpm_linux_bios_measurements
+  title: TPM 2.0 BIOS measurements exposed by Linux kernel
+  license: CC0-1.0
+  endian: le
+doc: |
+  Linux kernel exposes the TPM 2.0 eventlog in a dedicated file,
+  /sys/kernel/security/tpm0/binary_bios_measurements.
```

Comment:
Can I ask you to copy the excellent description in your PR into this `doc`? Almost nobody will see the PR, but everybody will have a chance to read that from here.

Reply:
I agree. Done, https://github.com/kaitai-io/kaitai_struct_formats/pull/256/commits/3d177f0fb15c8e244acecda91a84a66f5ab2a2fc